### ISSUE TITLE
CircleCI v1 is being deprecated August 31, 2018

### ISSUE THREAD

**Issue by shiftkey** [desktop/desktop#4135]
[Just announced today](https://circleci.com/blog/sunsetting-1-0/), our CircleCI builds will stop working  at the end of August, and they are encouraging everyone to migrate to the new [v2](https://circleci.com/docs/2.0/) definitions.

A good starting point for a reference config would probably be https://github.com/CircleCI-Public/circleci-demo-react-native/blob/master/.circleci/config.yml and this section: https://github.com/CircleCI-Public/circleci-demo-react-native/blob/master/.circleci/config.yml#L32-L56

**Comment by gengjiawen** [comment]
So any specific reason both circle ci and travis ci?

**Comment by shiftkey** [comment]
> So any specific reason maintain both circle ci and travis ci?

Not really, just haven't had a chance to evaluate CircleCI's Linux support

**Comment by gengjiawen** [comment]
I recently has some work related to circle ci. I begin to understand why many repo migrate from travis to circle ci. 
Some advantage:
* cache support (demo: https://github.com/vuejs/vue/blob/dev/.circleci/config.yml#L13)
* debug your build (This is extremely handy. Travis is in beta, and you have to email them to get support)
* better docker support
* parallel build

Some drawback:
* Linux only has 4GB memory compare to travis plan.

I think linux support on cirrcle ci is fine. But docker support may not an option because the link problem in docker.
